gradlew

```diff
@@ -26,18 +26,21 @@ cd "$SAVED" >/dev/null
 
 APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
+# Set the gradle home to be a local project directorys
+GRADLE_USER_HOME="$APP_HOME/build/gradle-home"
+echo "Setting the GRADLE_USER_HOME to $GRADLE_USER_HOME \n(needed for the creation of the offline repository)\n"
```

Comment:
This doesn't seem to do the trick anymore.. I tried using -Dgradle.user.home near the end of the script instead